flambe/runnable/runnable.py

```diff
@@ -27,6 +27,19 @@ class Runnable(MappedRegistrable):
     def __init__(self, **kwargs) -> None:
         self.config = configparser.ConfigParser()
         self.extensions: Dict[str, str] = {}
+        self.content: Optional[str] = None
+
+    def inject_content(self, content: str) -> None:
```

Comment:
in theory you should be able to just dump the experiment to YAML right?

Reply:
Yes, but that would add the default parameters. I want to display the exact config the user used.